docs/lago-tutorial/scenarios/jenkins/deployment-scripts/vm2-slave-deploy-script.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash -xe
+yum install -y java-1.8.0-openjdk-devel.x86_64 \
+  net-tools.x86_64
+
+# As this is the same as vm1, you can use the same script, or soft-link from
+# one to the other
```

Comment:
I removed the comment.
I prefer to keep one script for each vm, so the user can use them as a template for his uses, without changing anything in the init file.